libs/product/src/facades/helpers.ts

```diff
@@ -0,0 +1,19 @@
+import { DaffPriceRange } from '../models/prices';
+
+/**
+ * Returns whether a DaffPriceRange has a discount.
+ * @param priceRange a DaffPriceRange
+ */
+export const productPriceRangeHasDiscount = (priceRange: DaffPriceRange): boolean => {
```

Comment:
These aren't showing up in the dgeni docs, do you know how to get dgeni to pick them up? It might not be in the scope of this PR.

Reply:
No, I'm not sure. We also need to figure out how to get the selector docs to work. Definitely out of scope though